templates/bridge-infra/agglayer-config.toml

```
@@ -14,7 +14,8 @@ mock-verifier = {{.mock_verifier}}
 1 = "{{.zkevm_l2_sequencer_address}}"
 
 [rpc]
-port = {{.agglayer_port}}
+grpc-port = {{.agglayer_grpc_port}}
+readrpc-port = {{.agglayer_readrpc_port}}
```

This will probably fail for both fork 9 and 11 since we need to rely on an older version of the agglayer (agglayer:0.2.0-rc.19). We need to support both 0.2.0 and 0.3.0 versions. To do this you can have something like the following:

```suggestion
{{if eq .agglayer_version "0.2" }}
port = {{.agglayer_port}}
{{else}}
grpc-port = {{.agglayer_grpc_port}}
readrpc-port = {{.agglayer_readrpc_port}}
{{end}}
```

You would need to compute `.agglayer_version` from the `agglayer_image` variable and pass it to the template. This agglayer version variable would also be used to determine the ports of the service.